dvc/utils/__init__.py

```diff
@@ -34,52 +36,47 @@
 LOCAL_CHUNK_SIZE = 1024 * 1024
 LARGE_FILE_SIZE = 1024 * 1024 * 1024
 LARGE_DIR_SIZE = 100
+CHECKSUM_JOBS = max(1, min(4, cpu_count() // 2))
```

Comment:
This default looks ok, thought I would probably introduce paramterization through config, not necessarily in this change.

Reply:
Yes, I already have the config code prepared. It can't be used directly in `utils`, because `config` imports `utils`, so there would be a circular import, but we should be able to load the config in the caller of `file_md5`.

Reply:
I think it should be moved out of utils, especially from __init__ now that it have a sufficiently sophisticated logic, configurable constants, etc.